docs/UserGuide.md

```diff
@@ -246,7 +292,28 @@ When the command fails:
 
 ### <u>Category 3 - Find contacts</u>
 
+### Search: `search [/field]`
+
+You can use the `search` command to search for contacts by their name if you wish to quickly reference a particular contact’s details.
+
+Format: `search /name [name]`
+
+Example usage:
+* `search /name Jack`
+* `search /name Kai Jie`
+
+Parameters:
+* `[name]` is the name of the contact, or a part of it
+
+Expected output when the command succeeds:
```

Comment:
For consistency, this header can be changed to "When the command succeeds" to match with other commands. The same for when the command fails below